from google.api_core.exceptions import ResourceExhausted
import google.api_core
import google.auth
